submissions/NastjonkaK/OOP_exercise/index.js

```diff
@@ -3,82 +3,95 @@ class Creature {
     this.name = name,
     this.saying = saying
   }
+
+  introduction() {
```

Comment:
Change naming of your methods. It's a best practice to actually tell what the function or method is doing by giving the function name a verb as prefix. This will help to made the method name more self-descriptive.